<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchWindow</class>
 <widget class="QWidget" name="SearchWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../bin/images.qrc">
    <normaloff>:/images/search.png</normaloff>:/images/search.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Поиск</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolButton" name="btnCatalogsSelect">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/images/catalog.png</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnBooksSelect">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/images/book.png</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtSearch">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnFind">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/images/search_1.png</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnShow">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>31</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1280</width>
        <height>274</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="../../bin/images.qrc">
     <normaloff>:/images/search.png</normaloff>:/images/search.png</iconset>
   </property>
   <property name="text">
    <string>Поиск по текстам</string>
   </property>
   <property name="toolTip">
    <string>Поиск по текстам</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="action_books_select">
   <property name="icon">
    <iconset resource="../../bin/images.qrc">
     <normaloff>:/images/book.png</normaloff>:/images/book.png</iconset>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="toolTip">
    <string>Выбор книг для поиска</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset resource="../../bin/images.qrc">
     <normaloff>:/images/font.png</normaloff>:/images/font.png</iconset>
   </property>
   <property name="text">
    <string>Выбор шрифта</string>
   </property>
   <property name="toolTip">
    <string>Выбор шрифта</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="action_catalogs_select">
   <property name="icon">
    <iconset resource="../../bin/images.qrc">
     <normaloff>:/images/catalog.png</normaloff>:/images/catalog.png</iconset>
   </property>
   <property name="text">
    <string>Выбор каталогов для поиска</string>
   </property>
   <property name="toolTip">
    <string>Выбор каталогов для поиска</string>
   </property>
  </action>
  <action name="actionShow">
   <property name="icon">
    <iconset resource="../../bin/images.qrc">
     <normaloff>:/images/list.png</normaloff>:/images/list.png</iconset>
   </property>
   <property name="text">
    <string>Show</string>
   </property>
   <property name="toolTip">
    <string>Список оставшихся в окне цитат одним текстом</string>
   </property>
  </action>
  <action name="actionQuoteStorage">
   <property name="text">
    <string>QuoteStorage</string>
   </property>
   <property name="toolTip">
    <string>Работа с цитатами</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../bin/images.qrc"/>
 </resources>
 <connections/>
</ui>
